app/adminPage.html

```diff
@@ -74,6 +74,21 @@
         </div>
     </div>
 
+<div class="modal" id="delete_user_modal" tabindex="-1" role="dialog" data-toggle="modal">
```

Comment:
all good except: out of the four options you get in the admin page, "edit user" allows you to close by clicking outside the modal, whereas "delete user" does not. "edit user" loses its shadow when you click outside the modal, but does not close? don't know if relevant to your story but if it's an easy fix would be good for consistency.

Reply:
Have removed all click events from div overlay. All admin page modals must now be closed using the close cross or included buttons.